### DISCUSSION TITLE
x2 omni support

### DISCUSSION THREAD

**Discussion by fryc88** [MVladislav/bumper#56] [General]
Any ideas if x2 omni is working or will be working?

**Comment by MVladislav** [comment]
Hi, since I don’t have an "Omni X2" model, I am unable to check what isn't working. 
I’ve updated the project today with a newer Python version, which also includes an updated product list for Deebots. 
It's possible that the issue occurred because your version wasn't listed in the JSON file. 
Please note that there are multiple versions of the "Omni X2" model.

You can check the updated list here:
https://github.com/MVladislav/bumper/blob/dev/bumper/web/plugins/api/pim/productIotMap.json

**Reply by vordenken** [reply]
Hi @MVladislav, I got an X2 Omni and found your project. I successfully got the app connected to bumper but not the robot. Is there any debugging I can do? Is it because of cert pinning on the robot? 

Thanks for your work btw!

**Comment by fryc88** [comment]
Thanks for replying. I guess i can give it a go then. Mine is definitely not 'pro', it just says deebot x2 omni

**Reply by siccovansas** [reply]
Did you get the x2 to work with bumper?

**Comment by fryc88** [comment]
At the end I haven't tried it.
